apps/10address-book-ajax/script.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
 $(document).ready(function() {
-    // You code here
-});
\ No newline at end of file
+    $.ajax('https://reqres-api.herokuapp.com/api/users', {
+      success: function(users){
+        users.forEach(function(user){
+          var $tr = $('<tr></tr>');
+          $tr.append('<td>'+ user.id + '</td>');
+          $tr.append('<td>'+ user.first_name + '</td>');
+          $tr.append('<td>'+ user.last_name + '</td>');
+          var $a = ('<td><a href="#" data-id=" ' + user.id + '">view</a></td>');
+          $tr.append($a);
+          $('tbody').append($tr);
+        });
+
+          $('a').on('click',function(){
```

Comment:
You forgot to pass in 'event' as a parameter here, thus event.preventDefault will run as undefined. Ex:

$('a').on('click', function(event){
        event.preventDefault();
        ...
}